models/user.go

```diff
@@ -19,7 +19,10 @@ func CreateUser(u *User) (id int64, err error) {
 
 func (u *User) IsExistingUser() bool {
 	o := orm.NewOrm()
-	err := o.Read(u, "Email")
+	var user User
+
+	err := o.QueryTable(User{}).Filter("email__iexact", u.Email).One(&user)
```

Comment:
@junan By doing this `__iexact`, why don't we try on with lowercase the email before save and query?
The benefit of doing this:
1. clean email in the database
2. we don't have to suffix with `__iexact` 🤣

Or the other way is set it on migration file as @carryall did 🤣

Reply:
Regarding store the email in lowercase, In this [answer](https://stackoverflow.com/a/1503679), it suggested to store as it it.

> Or the other way is set it on migration file as @carryall did

I have  beenusing Beego automatic table generation with `RunSyncdb ` and I don't want to switch back to manual migration where I need create all the table from migration again 😄 

For me using Beego's `__iexact` is ok. What do you think?